deployment/terraform/modules/vpc/vars.tf

```diff
@@ -2,10 +2,9 @@ variable "vpc_cidr_block" {}
 variable "private_cidr_block_a" {}
 variable "private_cidr_block_b" {}
 variable "public_cidr_block" {}
-variable "availability_zone" {}
-variable "availability_zone_a" {
-  default = "us-east-1a"
-}
-variable "availability_zone_b" {
-  default = "us-east-1b"
+variable "availability_zone_a" {}
+variable "availability_zone_b" {}
```

Comment:
Is the availability zone now something that gets specified on the command line when invoking terraform, or does it come from somewhere else?